custom/icds_reports/utils/aggregation_helpers/distributed/bihar_api_demographics.py

```diff
@@ -109,7 +109,15 @@ def aggregation_query(self):
             ('date_last_private_admit', 'person_list.date_last_private_admit '),
             ('date_return_private', 'person_list.date_return_private'),
             ('out_of_school_status', 'person_list.is_oos'),
-            ('last_class_attended_ever', 'person_list.last_class_attended_ever')
+            ('last_class_attended_ever', 'person_list.last_class_attended_ever'),
+            ('has_bank_account', 'CASE WHEN person_list.bank_account_number is not null THEN 1 ELSE 0 END'),
+            ('age_marriage', 'person_list.age_marriage'),
+            ('last_referral_date', 'person_list.last_referral_date'),
+            ('referral_health_problem', 'person_list.referral_health_problem'),
+            ('referral_reached_date', 'person_list.referral_reached_date'),
+            ('referral_reached_facility', 'person_list.referral_reached_facility'),
+            ('migrate_date', 'migration_tab.migration_date'),
+            ('is_alive', 'CASE WHEN person_list.date_death is null THEN 1 ELSE 0 END')
```

Comment:
Also the check out not only limit till `null` but will also include `0`. Something like is `CASE WHEN date_death IS DISTINCT FROM 1 THEN 1 else 0 END`

Reply:
moved it to view. And why to check distinct from 1 as it is a date